package.json

```diff
@@ -246,7 +246,9 @@
     "yauzl": "2.7.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.3.4",
     "@babel/parser": "^7.3.4",
+    "@babel/polyfill": "^7.2.5",
```

Comment:
@spalger cannot find where added `@babel/polyfill` in `kbn-test`. Seems `useBuiltIns: 'entry',` doesn't work there. Does it?

Reply:
Hmm, well kbn-test is always executed in place, so it's probably borrowing the dependencies from kibana, and since `useBuiltIns: 'entry'` replaces `@babel/polyfill` with individual requires for `core-js/modules/{feature}` it's probably just borrowing the core-js version required by Kibana... Not the best strategy, but I guess that's why it works. That said, we don't actually require `@babel/polyfill` in any babel 7 code, yet. @mistic is working on converting the root babel usages to babel 7 and at that time we'll be moving from `babel-polyfill` to `@babel/polyfill` in `src/setup_node_env/babel_register/polyfill.js`

Reply:
if package relies on environment to provide polyfilled runtime we might want to use` useBuiltIns: false` https://babeljs.io/docs/en/babel-preset-env#usebuiltins-false